analyze.go

```diff
@@ -7,6 +7,25 @@
 
 package crossplane
 
+// Update for headersmore
+//go:generate sh -c "sh ./scripts/generate/generate.sh --url https://github.com/openresty/headers-more-nginx-module.git --config-path ./scripts/generate/configs/headersmore_config.json > ./analyze_headersMore_directives.gen.go"
+
+// Update for njs
+//go:generate sh -c "sh ./scripts/generate/generate.sh --url https://github.com/nginx/njs.git --config-path ./scripts/generate/configs/njs_config.json > ./analyze_njs_directives.gen.go"
+
+// Update for OSS, filter in config is the directives not in https://nginx.org/en/docs/dirindex.html but in source code.
+// Override in config is for the "if" directive. We create a bitmask ngxConfExpr for it in crossplane, which is not in source code.
```

Comment:
nit: do these comments about filter and override for each module make more sense in the corresponding json files? Or does that mess with the generator?

Reply:
I also wanted in this way but sadly... json doesn't provide comments. I could only comment by adding keys like "filter_comment":"....." But seems a bit weird. 
I thought not deal breaker? Normally user will go to `analyze.go` to add & review `//go:generate` comments first.

Reply:
Yeah not a deal breaker :) comments in analyze.go works!